Conan.VisualStudio/Utils.cs

```diff
@@ -0,0 +1,56 @@
+using System;
```

Comment:
I think this should go to the core project, as it doesn't depend on Visual Studio interfaces

Reply:
It needs `using Microsoft.VisualStudio.Threading`, don't know why 🤷‍♂